ui/main/src/app/views/customCardList/ResultTable.ts

```diff
@@ -112,6 +112,14 @@ export class ResultTable {
                             flex: column.flex
                         });
                         break;
+                    case FieldType.HTML:
+                        agGridColumns.push({
+                            field: column.field,
+                            headerName: column.headerName,
+                            type: 'html',
+                            flex: column.flex
+                        });
```

Comment:
Modify unit test "Should get columns definition for ag-grid" to have an html type in the list to be tested